_caster.py

```diff
@@ -36,7 +36,7 @@ def _wait_for_wsr_activation():
 from castervoice.lib import control
 _NEXUS = control.nexus()
 
-from castervoice.apps import *
+from castervoice.apps import __init__
```

Comment:
apps/__init__ has different functionality in this patch. It now pulls in all the files looking under the directory /apps. If you call `from castervoice.apps import *` instead it will error out (I believe because it tries to import twice).

Reply:
Remind me of the downside of your current implementation? 
The reason we have it set up this way as it checks for errors in Python files and excludes them loading of caster. Otherwise it crashes the whole system.

Reply:
No real downside that I'm aware of. Still verifies the rules and will only fail in a single rule at a time. It just is a little strange that I had to change it. I'm thinking it's because for example chrome.py imports browser_shared_commands.py which means if we used a * here we would import it twice, but not really sure. I could test it more and try to "fix" it but not sure it's worth it.